app/javascript/article-form/articleForm.jsx

```diff
@@ -265,10 +265,10 @@ export class ArticleForm extends Component {
     });
   };
 
-  toggleModal = (showModal) => {
+  showModal = (isOpen) => {
     if (this.state.edited) {
       this.setState({
-        showModal,
+        showModal: isOpen,
```

Comment:
refactor (nitpick): Maybe call this state property `isModalOpen` so we don't have `showModal` in state and as a method.